app/pages/lab/workflow-viewer/answers.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+Sty = require './jsplumb-style'
```

Comment:
Generally speaking, it's not a great idea to abbreviate variable names outside of the most obvious cases, like array indexes. Full names act as great inline documentation for the next dev to come along. Could you go through the PR and convert shorthand names into their full name, e.g. `Sty` to `JSPlumbStyle`, `wf` to `workflow`, `pos` to `position`, etc.